src/index.tsx

```diff
@@ -63,6 +63,12 @@ if ((!navigator.onLine || process.env.NODE_ENV === 'development') && localStorag
   initApp()
 }
 
+// disable console log for prod
+if (process.env.NODE_ENV !== 'development') {
+  /* eslint-disable */
+  console.log = () => {}
```

Comment:
```suggestion
  const consoleMock = () => {}
  console.log = consoleMock;
  console.error = consoleMock;
  console.warn = consoleMock;
```

How about this so most common console methods are covered?

Reply:
@grgur Updated!

Reply:
for some reason that i am investigating these changes are not good for ci circle gimbal https://app.circleci.com/pipelines/github/moduslabs/modir/478/workflows/0210b794-dbd7-4711-a65f-52c2a410d6bf/jobs/1669

Reply:
warn is not quite good for gimbal, I am not so familiar with this internal process and logs are not so helpful , any recommendation ?